### DISCUSSION TITLE
is it possible to add collapse sections (and nested) to steps? test? (python + pytest)

### DISCUSSION THREAD

**Discussion by EitanBe** [allure-framework/allure2#1345] [Questions & Support] [answered]
is it possible to add collapse sections (and nested) to steps? test? (python + pytest)
thanks!!!!

**Comment by Avaddon-tfa** [comment, marked as answer]
Hi there! 

Not sure about collapsed sections, but nested steps are supported. 
Take a look at the example [here](https://github.com/allure-examples/allure-examples/blob/master/allure-python/allure-python-pytest/test/allure_step_test.py).

If I got you wrong, could you please specify your question?

**Reply by EitanBe** [reply]
@Avaddon-tfa Thank you so much you find the time to answer. 
In the sub-test, I have a JSON, and I would like this kind of info to collapse it and not all the INFO.

**Reply by Avaddon-tfa** [reply]
Please, let me ask you to do us a favor. 
Could you provide a repo with an example of a test with such a JSON in a sub-test?

An example is a good thing to investigate so you'd really help us doing so.

**Comment by boris-becker-p** [comment]
Is allure has option to attach  pytest logs to steps?

**Reply by baev** [reply]
please use separated discussion for asking questions
